src/commands.py

```diff
@@ -105,7 +105,7 @@ class Commands(Enum):
         description="Adds an address to a contact.",
         run=add_address,
         args_len=2,
```

Comment:
in this case must be -1
please double check

Reply:
def add_address(args: list[str], book: "AddressBook") -> None:
    """Adds an address to a contact in the address book.
    param: args: List with 2 values: name and address.
    param: book: AddressBook object to modify.
    return: str: Result message.
    """
    name = args[0]
    address = " ".join(args[1:])
    record = book.find(name)
    if not record:
        raise NotFoundWarning(f"Contact '{name}' not found")

    record.add_address(address)
    print_to_console("Address added.", style=OutputStyle.SUCCESS) add_address(contact, Kyiv Bandery avenue 1 app 13)

Reply:
do i have to change here ---->param: args: List with 2 values: name and address--->to list with any quantity where the first is contact name the rest is address